src/components/BAccordion/BAccordion.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <section :id="_uid">
+    <button
+      :id="`${_uid}-heading`"
+      class="header"
+      :aria-expanded="expanded"
+      :aria-controls="`${_uid}-content`"
+      aria-disabled="true"
+      role="heading"
+      type="button"
+      tabindex="0"
+      @click="toggleExpansion"
+      @keydown.enter.prevent
+      @keyup.enter.prevent="toggleExpansion"
+      @keydown.space.prevent
+      @keyup.space.prevent="toggleExpansion"
+    >
+      {{ label }}
+      <span
+        :class="{
+          chevron: true,
+          'ion-chevron-down': expanded,
+          'ion-chevron-up': !expanded
+        }"
+      >
+      </span>
+    </button>
+    <transition name="fade">
+      <div
+        v-if="expanded"
+        :id="`${_uid}-content`"
+        class="bordered"
+        role="region"
+        :aria-labelledby="`${_uid}-heading`"
+      >
+        <slot></slot>
+      </div>
+    </transition>
+  </section>
+</template>
+
+<script>
+import events from '../../event-bus'
+
+export default {
+  name: 'b-accordion',
+  props: {
+    label: {
+      type: String,
+      required: true
+    }
+  },
+
+  created () {
+    events.$on('collapse', this.collapse)
+    events.$on('expand', this.expand)
+  },
+
+  beforeDestroy () {
+    events.$off('collapse', this.collapse)
+    events.$off('expand', this.expand)
+  },
+
+  data: function() {
+    return {
+      expanded: true
+    }
+  },
+
+  methods: {
+    toggleExpansion () {
+      this.expanded = !this.expanded
+    },
+    collapse () {
+      if (this.expandable) {
+        this.expanded = false
+      }
+    },
+    expand () {
+      this.expanded = true
+    }
+  },
+
+  inject: {
+    parentScope: {
+      default: null
+    }
+  }
```

Comment:
What's the purpose of this inject?  It doesn't get used anywhere that I can see.